Automattic/woocommerce-payments
# Added payment methods cache cleaning when account is updated via webhook

Fixes #4138

#### Changes proposed in this Pull Request

When the account is updated via webhook, the webhook triggers a function that clears account cache data. In addition to the previous described functionality, new code is added that schedules an action that will clear cache for stored payment methods





#### Testing instructions

1. Make sure to have WordPress account with at least one payment methods that is added within 24 hours.
2. Buy random product, so we are certain that stored payment method is working, and it is cached.
3. Update account data via Stripe or re-onboard
4. Open wp_options table, and you should not see any transients that start with wcpay_pm_[CUSTOMER_ID]

-------------------


**Post merge**
